.github/workflows/on-push.yml

```diff
@@ -290,7 +290,7 @@ jobs:
     - name: Run slow tests
       if: matrix.task == 'slow'
       run: |
-        poetry run pymedphys dev tests -v --run-only-slow
+        poetry run pymedphys dev tests -v --slow
```

Comment:
Where can I find that the changed name for the option is implemented?
I'm struggling to find it in this PR.

Reply:
Yeah, I added thos configuration options a little while back. I didn't remove the alternative, just added an easier flag. It was bugging me during usage, I kept mixing up "run-only-slow" with "only-run-slow".

Here's where the options are configured:

https://github.com/pymedphys/pymedphys/blob/15a79cb3036e66ad111dc8a66825245ad5714eb9/pymedphys/conftest.py#L9-L34